tool.html

```diff
@@ -60,8 +60,11 @@ <h3 class='open-sans-sans-serif-text'>Download MetaSUB data</h3>
 			<h3 class='align-text-center open-sans-sans-serif-text'>Entropy Analysis</h3>
 		</div>
 		<div class='vertical-separator-from-top-50' ng-show='myCtrl.showAnalysis'>
-			<div class='align-text-center italic-font-size-25' ng-if='!myCtrl.showEntropy'>Analyzing {{myCtrl.fileName}} ...</div>
-			<div class='align-text-center boxed bold-font-size-30 orange-text' ng-if='myCtrl.showEntropy'>Overall Entropy(Approx) = {{myCtrl.entropy | number : 4}}</div>
+			<div class='align-text-center italic-font-size-25 orange-text' ng-if='!myCtrl.showEntropy'>
+				<div class="loader"></div>

```

Comment:
@MAZHARMIK - This is looking great! Just one thing, the style = `loader` is missing from this PR (I also checked your other PR - https://github.com/gbelwariar/Microbiome-Diversity-Inspector/pull/96, and it was missing from there as well). You can add the change in `tool.css` to the same branch as discussed today :)